feat: Install gandalf, salt-wrapper for infra

tomashley opened the pull request:
* Installs Gandalf for use as Salt v2 wrapper
* Gandalf is decentralizing salt runs

darora committed:
chore: strip out unnecessary dependencies from QEMU builds

Also upgrades to Alpine 3.22.

tomashley committed:
feat: Install gandalf, salt-wrapper for infra

Crispy1975 commented on ansible/vars.yml:
```diff
@@ -57,3 +57,5 @@ adminmgr_release: 0.25.1
 vector_x86_deb: "https://packages.timber.io/vector/0.22.3/vector_0.22.3-1_amd64.deb"
 vector_arm_deb: "https://packages.timber.io/vector/0.22.3/vector_0.22.3-1_arm64.deb"
+
+gandalf_release: 1.4.18
```
`1.4.21`

Crispy1975 replied:
`v1.4.25` now.

tomashley committed:
Merge branch 'develop' into tom/ansible/gandalf

tomashley committed:
chore: version bumps

tomashley committed:
chore: version bumps

pcnc commented on ansible/vars.yml:
```diff
@@ -10,8 +10,8 @@ postgres_major:
 # Full version strings for each major version
 postgres_release:
   postgresorioledb-17: "17.0.1.093-orioledb"
```
Oriole needs a bump as well

tomashley replied:
bumped

tomashley committed:
bumping oriole build also

tomashley committed:
fix: create the Gandalf group

tomashley committed:
fix: implicitly create the gandalf group

tomashley committed:
fix: correct gandalf archive path

tomashley committed:
fix: unpack gandalf build correctly

tomashley committed:
fix: reorganise order for gandalf installs

tomashley committed:
fix: point to the correct path

samrose committed:
fix: pin version of libpq to 17.x (#1649)

* fix: pin version of libpq to the major version installed

* fix: the variable is postgresql_major_version

* fix: using the correct var

* fix: default to postgres 15 major version

* fix: default to 15

* fix: supposedly main contains all releases for libpq5

* fix: syntax

* fix: different syntax

* fix: rely on ppa

* fix: ppa removal

* fix: change to trigger workflow

* fix: adding a test to see postgres version

* fix: testing for failure related to libpq5 version issue

* test: include read-only test for postgrest

* test: reduce log access time

* test: make sure to fail if version wrong

* chore: bump vars

darora committed:
chore: disable auth config reloading within qemu artifact

The QEMU artifact is not running with a filesystem that supports the
inotify API, resulting in spurious error messages, and broken config
reloading.

darora committed:
ci: no longer build Oriole QEMU artifact

Said artifact is no longer being used. If needed in the future, it can
be trivially reintroduced, but in the meantime there's no reason to
expend resources on them.

samrose reviewed (approved):
Approving although it may be worth thinking about how to test this out in a running instance.

Crispy1975 committed:
chore: bump vars due to conflicts

Crispy1975 committed:
Merge branch 'develop' into tom/ansible/gandalf

Crispy1975 committed:
tests: update user and group test adding gandalf

Crispy1975 committed:
chore: add gandalf user to user list
